pkg/provider/aws.go

```diff
@@ -276,3 +282,7 @@ func GetAwsAccount() (string, error) {
 	}
 	return res.Account, nil
 }
+
+func (aws *AWSProvider) KeyID() string {
```

Comment:
I don't believe this should be set on the cloud provider actually (I mean we can but it should be orthogonal to it).  Is there a way we can remove the dependency there?

Reply:
we can remove it, especially when we would like to use a new file outside the encrypted list